lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip-tabbed/search-facet-chip-tabbed.component.spec.ts

```diff
@@ -0,0 +1,232 @@
+import { By } from '@angular/platform-browser';
+import { SearchQueryBuilderService } from '../../../services/search-query-builder.service';
+import { SearchFilterList } from '../../../models/search-filter-list.model';
+import { SearchFacetChipTabbedComponent } from './search-facet-chip-tabbed.component';
+import { TabbedFacetField } from '../../../models/tabbed-facet-field.interface';
+import { FacetField } from '../../../models/facet-field.interface';
+import { SearchFacetFiltersService } from '../../../services/search-facet-filters.service';
+import { SimpleChange } from '@angular/core';
+
+describe('SearchFacetChipTabbedComponent', () => {
+  let component: SearchFacetChipTabbedComponent;
+  let fixture: ComponentFixture<SearchFacetChipTabbedComponent>;
+  let queryBuilder: SearchQueryBuilderService;
+  let searchFacetService: SearchFacetFiltersService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+        imports: [
+            TranslateModule.forRoot(),
+            ContentTestingModule
+        ]
+    });
+    fixture = TestBed.createComponent(SearchFacetChipTabbedComponent);
+    component = fixture.componentInstance;
+    queryBuilder = TestBed.inject(SearchQueryBuilderService);
+    searchFacetService = TestBed.inject(SearchFacetFiltersService);
+    spyOn(queryBuilder, 'update').and.stub();
+
+    const facet1: FacetField = { type: 'field', label: 'field', field: 'field', buckets: new SearchFilterList() };
+    const facet2: FacetField = { type: 'field', label: 'field2', field: 'field2', buckets: new SearchFilterList() };
+
+    const tabbedFacet: TabbedFacetField = {
+        fields: ['field', 'field2'],
+        label: 'LABEL',
+        facets: {
+            field: facet1,
+            field2: facet2
+        }
+    };
+    component.tabbedFacet = tabbedFacet;
+    fixture.detectChanges();
+  });
+
+  function openFacet() {
+    const chip = fixture.debugElement.query(By.css('mat-chip'));
+    chip.triggerEventHandler('click', {});
+    fixture.detectChanges();
+  }
+
+  function getDisplayValue(): string {
+    return fixture.debugElement.query(By.css('.adf-search-filter-ellipsis.adf-filter-value')).nativeElement.innerText.trim();
+  }
+
+  function getTabs(): HTMLDivElement[] {
+    return fixture.debugElement.queryAll(By.css('.mat-tab-label-content')).map((element) => element.nativeElement);
+  }
+
+  function changeTab(tabIndex: number) {
+    getTabs()[tabIndex].click();
+    fixture.detectChanges();
+  }
+
+  function triggerComponentChanges() {
+    component.ngOnChanges({
+        tabbedFacet: new SimpleChange(null, component.tabbedFacet, false)
+      });
+    fixture.detectChanges();
+  }
+
+  function addBucketItem(field: string, displayValue: string) {
+    component.tabbedFacet.facets[field].buckets.items.push({ count: 1, label: displayValue, display: displayValue, filterQuery: '' });
+    triggerComponentChanges();
+  }
+
+    it('should display correct label for tabbed facet', () => {
```

Comment:
Please align all test cases, they seem not aligned looking at above functions. Functions are closed to left side

Reply:
Aligned